mappings/net/minecraft/block/CoralFanBlock.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS bnl net/minecraft/block/CoralTubeFanBlock
+CLASS bnl net/minecraft/block/CoralFanBlock
```

Comment:
Weren't these based on Mojang's registry IDs? Could you double-check with Blocks?

Reply:
I would assume they don't have individual classes for every type of coral fan? That wouldn't make sense